scripts/zones/Waughroon_Shrine/mobs/Osschaart.lua

```diff
@@ -0,0 +1,87 @@
+-----------------------------------
+-- Area: Waughroon Shrine
+--  Mob: Osschaart
+-- KSNM: Copycat
+-----------------------------------
+local twoHourAbility =
+{
+    xi.jsa.MIGHTY_STRIKES,
+    xi.jsa.HUNDRED_FISTS,
+    xi.jsa.BENEDICTION,
+    xi.jsa.MANAFONT,
+    xi.jsa.CHAINSPELL,
+    xi.jsa.PERFECT_DODGE,
+    xi.jsa.INVINCIBLE,
+    xi.jsa.BLOOD_WEAPON,
+    xi.jsa.FAMILIAR,
+    xi.jsa.SOUL_VOICE,
+    0, -- EAGLE_EYE_SHOT
```

Comment:
Lets put some kind of ID in here even if its not the correct one so he can still 2hr.

Reply:
Tested a few, and kindred doesn't cause pretzeling. looks really good.